contacts/style.css

```diff
@@ -0,0 +1,198 @@
+:root {
+    --main-background-color: #ffdd99;
+    --other-background-color: #336600;
+    --text-color: #ffdd99;
+    --nice-black-color: #001a13;
+    --pop-color: #15945f;
+}
+
+.align-center {
+    text-align: center;
+}
+
+html {
+    background-color: var(--main-background-color);
+}
+
+header {
+    width: 100%;
+    background-color: var(--other-background-color);
+    border: 2px solid black;
+    border-radius: 25px;
+    margin: 2px;
+}
+
+header h1 {
+    padding: 0px;
+    font-size: 3.5em;
+    color: var(--main-background-color)
+}
+
+nav {
+    margin: 2px;
+}
+
+nav ul {
+    background-color: var(--other-background-color);
+    margin: 0px;
+    padding: 10px;
+    overflow: hidden;
+    border: 2px solid black;
+    border-radius: 25px;
+}
+
+nav ul li {
+    display: inline;
+    font-size: 2em;
+    padding: 40px;
+    color: var(--text-color);
+}
+
+nav ul li:hover {
+    background-color: var(--nice-black-color);
+}
+
+nav ul li a {
+    color: var(--text-color);
+    text-decoration: none;
+}
+
+.separator {
+    display: inline;
+    font-size: 2em;
+    padding: 0px;
+    color: var(--text-color);
+}
+
+footer {
+    background-color: var(--other-background-color);
+    padding: 10px;
+    border: 2px solid black;
+    border-radius: 25px;
+    margin: 2px;
+}
+
+footer p {
+    padding: 50px;
+    margin: 20px;
+    text-align: right;
+    display: inline;
+    color: var(--text-color);
+}
+
+.person-field {
+    /* to do */
+    padding: 10px;
+    border: 2px solid black;
+    border-radius: 25px;
+    margin: 2px;
+}
+
+.person-field:hover {
+    background-color: var(--pop-color);
+    color: var(--text-color);
+    transform: scale(1.05);
+}
+
+.no-likes {
+    display: inline-block;
+    margin: 5px 10px 0px 10px;
+    padding: 15px 0px 0px 0px;
+    position: relative;
+    width: 40px;
+    height: 34px;
+    background: url(images/star.png);
+    background-size: 40px;
+    background-repeat: no-repeat;
+}
+
+.star {
+    position: relative;
+}
+
+.number {
+    position: relative;
+    text-align: center;
+    width: 100%;
+    top: 20%;
+    transform: translate(0%, -50%);
+}
+
+.avatar {
+    vertical-align: middle;
+    width: 30px;
+    height: 30px;
+    border-radius: 50%;
+    margin: 0px 10px 0px 16px;
+}
+
+.list {
+    list-style-type: none;
+    padding:  10px 10px 10px 10px;
```

Comment:
as far as i know this can be replaced by padding: 10px, if you have the same value for all